arborio/include/arborio/swcio.hpp

```diff
@@ -170,7 +171,7 @@ swc_data parse_swc(const std::string&);
 // each SWC record after the first: this record defines the tag and distal point
 // of the segment, while the proximal point is taken from the parent record.
 
-arb::segment_tree load_swc_arbor(const swc_data& data);
+arb::morphology load_swc_arbor(const swc_data& data);
```

Comment:
The comments above this function prototype and the ones below say `SWC records to a morphological segment tree`, which is now `SWC records to a morphology`.

Reply:
Done.